serial.go

```diff
@@ -91,8 +91,13 @@ type Config struct {
 	// CRLFTranslate bool
 }
 
+type ReadWriteFlushCloser interface {
+	io.ReadWriteCloser
+	Flush() error
+}
+
 // OpenPort opens a serial port with the specified configuration
-func OpenPort(c *Config) (io.ReadWriteCloser, error) {
+func OpenPort(c *Config) (ReadWriteFlushCloser, error) {
```

Comment:
OK, I have renamed the package to github.com/tarm/serial, changed the interface to return an opaque *Port, and recreated the old repo as goserial.

That should prevent people's code from being broken while still let us make changes going forward.

Can you please rebase you changes?